sql/session.go

```diff
@@ -43,6 +43,9 @@ const (
 	characterSetConnectionSysVarName = "character_set_connection"
 	characterSetResultsSysVarName    = "character_set_results"
 	collationConnectionSysVarName    = "collation_connection"
+
+	// InformationSchemaDatabaseName is the name of the information schema database.
+	InformationSchemaDatabaseName = "information_schema"
```

Comment:
`session.go` doesn't seem like a great fit for this constant (although I like it being in the `sql` package). Did we have an import cycle or something with keeping it in the `information_schema` package?

Reply:
Yea I was also wondering this does not seem like the best place to put this, but I could not find better place than this? :thinking:
Yep, import cycle happened.